### DISCUSSION TITLE
Using HyperHDR for native TV video, NF (preinst.) etc.

### DISCUSSION THREAD

**Discussion by Basiayo** [awawa-dev/HyperHDR#873] [Q&A] [answered]
Hello, 
I am currently planning on getting a new TV (Sony X90L) and wanted to make a test setup in advance. (Never done HyperHDR before)
Now, my problem is (might be the fact, that I am not a native speaker), that I could not find any clear information about getting HyperHDR running directly from movies etc. played on the TV or installed applications like Netflix directly (If i got it right and all guides are supposed to be for external HDMI sources) 

Would be a bummer if that's not working at all, as that's the only way i watch right now. 

If that's possible, what HW would I need to get it running? I wouldn't like to buy devices that end up being completely useless else. 

So, I know about the following stuff that i definetely need: 
ESP32 (got it already) 
LED Strip (obv.) 
Power Supply 
RPi4 (Not ordered yet, as I want to be sure that it actually is possible how I want to make it) 

Now about the stuff I am not really sure about:
USB Grabber
HDMI Splitter

Which of those would I actually need? (And if possible some recommendations of products available on the german market?)

Thanks in advance.

**Comment by awawa-dev** [comment, marked as answer]
Hi
> I could not find any clear information about getting HyperHDR running directly from movies etc. played on the TV or installed applications like Netflix directly (If i got it right and all guides are supposed to be for external HDMI sources)

Yes, unfortunately it's true. Corporations assumed from above that you did not have the right to the signal for which you paid and the display on the TV, and Intel implemented it. So we have this unfortunate HDCP and the only way to bypass it to have ambilight is to use an HDMI splitter and external video source. However, local playback is also protected by, for example, widevine, which does not allow you to even take a screenshot of the application when protected content is being played. Even if the OS is rooted.

> ESP32 (got it already)

if you dont need a Wifi connection then Pico with a level shifter is a better soluition

> USB Grabber
> HDMI Splitter

only if you have an external video source. Otherwise, there is no point in even starting to build a setup.

**Reply by Basiayo** [reply]
Thank's for the quick reply. 
What a bummer :/

So the last option would be getting an AVR with streaming options i guess.

**Reply by awawa-dev** [reply]
In this case, the most common choices for the external video source are PCs, Fire TV Stick, Apple TV, NVidia Shield, TV boxes etc

**Reply by Basiayo** [reply]
Alright, after some research I think that a FireTV Stick might be an option. 
Thanks again!
